addons/l10n_mx/migrations/2.2/post-migrate_update_tax.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+
+def migrate(cr, version):
+    # Create new accounts for each MX company if accounts with relevant codes aren't already present
+    # Create DIOT 2025 taxes dependent on these accounts
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    new_account_list = [
```

Comment:
```suggestion
    new_account_vals_list = [
```

It would also be good to create the new accounts with the correct XMLids, as if they were created by loading the template.

Reply:
I will have to check for if the XMLid is already present in this case, but I can make it work like that if it is not.